server/lib/deriva/provider.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+from urllib.parse import urlparse
+
+from ..data_map import DataMap
+from ..entity import Entity
+from ..bdbag.bdbag_provider import BDBagProvider
+
+from girder.plugins.oauth.providers.globus import Globus
+
+Globus.addScopes(['https://auth.globus.org/scopes/a77ee64a-fb7f-11e5-810e-8c705ad34f60/deriva_all'])
+
+
+class DerivaProvider(BDBagProvider):
+    def __init__(self) -> None:
+        super().__init__('DERIVA')
+
+    def matches(self, entity: Entity) -> bool:
+        return str(entity.getValue()).startswith('https://pbcconsortium.s3.amazonaws.com/')
```

Comment:
Consider putting this in a configuration setting instead of hardcoding ala https://github.com/whole-tale/girder_wholetale/blob/master/server/lib/dataverse/provider.py#L96?  This way we can add new DERIVA installations without re-deploying.